package.json

```diff
@@ -126,7 +126,6 @@
     "@testing-library/react-hooks": "^5.1.0",
     "@testing-library/user-event": "^13.0.7",
     "@web-stories-wp/e2e-tests": "*",
-    "@wordpress/babel-plugin-import-jsx-pragma": "^3.0.1",
```

Comment:
Why is this removed?

Reply:
Because it‘s not needed anymore because of the new JSX transform